Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM nvidia/cuda:11.0.3-cudnn8-devel-ubuntu18.04
 CMD nvidia-smi
 
-RUN apt-get update && apt-get install -y build-essential && apt-get -y install curl
-RUN apt-get -y install python3.8 python3-distutils && ln -s /usr/bin/python3.8 /usr/bin/python
+RUN apt-get update && apt-get install -y build-essential curl python3.7 python3.7-dev \
```

Comment:
Just curious, what's the reason for changing to python3.7?

Reply:
The reason is just the combination of dependencies. `ydata-synthetic` depends on `numpy~=1.19`, which is a `numpy` version we support only for `python < 3.8`. Because of this, if we use 3.8 we end up installing a numpy version which ydata would not be officially compatible with. Since the conflicts do not show up on 3.7 and we also support it, I decided to switch to it.